pkg/server/status.go

```
@@ -40,10 +41,14 @@ func (c *criContainerdService) Status(ctx context.Context, r *runtime.StatusRequ
 		Type:   runtime.NetworkReady,
 		Status: true,
 	}
+	// Check the status of the cni initialization
 	if err := c.netPlugin.Status(); err != nil {
```

Check `ErrCNINotInitialized`? If it is not the error, we should just return the error in message.

---

status only returns the not initialized error

---

OK. Then let's just keep your current code. :)